internal/commands/verify.go

```diff
@@ -16,12 +16,45 @@ import (
 	"github.com/spf13/viper"
 )
 
+const verifyDesc = `
+This command executes Rego unit tests.
+
+Any file with a '_test' postfix and '.rego' extension will be compiled and 
+any Rego tests inside of them will be executed. For more information on how 
+to write tests check out the Rego testing documentation: 
+https://www.openpolicyagent.org/docs/latest/policy-testing/.
+
+The policy location defaults to the policy directory in the local folder.
+The location can be overridden with the '--policy' flag, e.g.:
+
+	$ conftest verify --policy <my-directory>
+
+Some policies are dependant on external data. This data is loaded in seperatly 
+from policies. The location of any data directory or file can be specified with 
+the '--data' flag. If a directory is specified, it will be recursively searched for 
+any data files. Right now any '.json' or '.yaml' file will be loaded in 
+and made available in the Rego policies. Data will be made available in Rego based on 
+the file path where the data was found. For example, if data is stored 
+under 'policy/exceptions/my_data.yaml', and we execute the following command:
+
+	$ conftest verify --data policy
+
+The data is available under 'import data.exceptions'.
+
+As with the test command, verify supports the '--output' flag to specify the type, e.g.:
+
+	$ conftest verify --output json
+
+For a full list of available output types, see the of the '--output' flag.
+`
```

Comment:
we have also `--trace` flag for `verify`. It would be great also it's documented.